docs/amphtml/get-started.md

```diff
@@ -11,22 +11,19 @@ localization_priority: Priority
 
 ## Author AMP for Email code
 
-1. Read the [AMP for Email specification](https://amp.dev/documentation/guides-and-tutorials/learn/amp-email-format) to understand the fundamentals. 
-1. If this is the first time you create an AMP email we recommend you to start with this [Tutorial](https://amp.dev/documentation/guides-and-tutorials/start/create_email/?format=email)
-1. Develop your code using [AMP Components](https://amp.dev/documentation/components/?format=email) and test using the [AMP Playground](https://playground.amp.dev/?runtime=amp4email). 
-
+1. Read the [AMP for Email specification](https://amp.dev/documentation/guides-and-tutorials/learn/amp-email-format) to understand the fundamentals.
+1. If this is your first time creating an AMP email we recommend you to start with this [tutorial](https://amp.dev/documentation/guides-and-tutorials/start/create_email/?format=email)
+1. Develop your code using [AMP Components](https://amp.dev/documentation/components/?format=email) and test using the [AMP Playground](https://playground.amp.dev/?runtime=amp4email).
 
 ## Test AMP emails in Outlook.com
-When developing AMP emails, you can use your own Outlook.com account to test them. 
 
-1. Register the sender address with Outlook. During development you can self-register. See [details](register-outlook.md)
-1. Send AMP email to your Outlook.com mailbox. 
+When developing AMP emails, you can use your own Outlook.com account to test them.
+
+1. [Register the sender address](register-outlook.md) with Outlook. During development you can self-register.
```

Comment:
Should be clear here that you can only send to yourself. I know you say it elsewhere, but we had this same type of confusion with AM :). It pays to be overly explicit.

Reply:
You can actually send to anyone you want. Wheter that email will render the AMP part depends on the client. We will allow any mailbox to individually whitelist which senders they allow.